src/main/java/org/minions/devfund/chelo/MovieDatabase.java

```diff
@@ -0,0 +1,186 @@
+package org.minions.devfund.chelo;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * MovieDatabase class.
+ */
+public class MovieDatabase {
+    private ArrayList<Movie> movieList;
+    private ArrayList<Actor> actorList;
+
+    /**
+     * MovieDatabase constructor.
+     */
+    MovieDatabase() {
+        movieList = new ArrayList<>();
+        actorList =  new ArrayList<>();
+    }
+
+    /**
+     * Getter for MovieList.
+     * @return movie list
+     */
+    public ArrayList<Movie> getMovieList() {
+        return movieList;
+    }
+
+    /**
+     * Getter for ActorList.
+     * @return actor list
+     */
+    public ArrayList<Actor> getActorList() {
+        return actorList;
+    }
+
+    /**
+     * This method takes in the name of a movie that is not currently in the database.
+     * Along with a list of actors for that movie.
+     * @param name movie name
+     * @param actors actors list
+     */
+    void addMovie(final String name, final String[] actors) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 16 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3776)